addons/sys_gestures/XEH_postInit.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+if (is3DEN || {!hasInterface || {!(GVAR(gesturesEnabled))}}) exitWith {};
+if (!isClass (configFile >> "CfgPatches" >> "ace_common")) exitWith {}; // No ACE exit
+
+GVAR(vestRadioArr) = parseSimpleArray (GVAR(vestRadios));
```

Comment:
I don't like how these settings are just a raw user input string.
Any mistake in input and it fails.
```
 Error in expression <s);
acre_sys_gestures_headsetRadioArr = parseSimpleArray (acre_sys_gestures_head>
   Error position: <parseSimpleArray (acre_sys_gestures_head>
  Error Generic error in expression
File idi\acre\addons\sys_gestures\XEH_postInit.sqf..., line 9
```

Reply:
Suggestions?

Reply:
Should have thought of that earlier, but what about an API function that we can validate? It's not as simple as CBA Settings, but way less chance of people breaking it.

Reply:
I wasn't really a big fan of this being a setting in the first place. I see two options here:

1. We dump one of the animations and just have one for all radios regardless (cheap and easy but I don't like it).
2. We have a function to build the arrays based on all the radios loaded as people might use custom ones. We then assign the in to arrays based on short range/long range (or some other attribute).

I'd prefer to keep both anims, it's nice to have a bit of variety.

Reply:
- vest radios: [play vest anim (default), play headset anim, play no anim]
- headset radios: [play vest anim, play headset anim (default),, play no anim]
`if (isNil QGVAR(headsetRadios)) then { GVAR(headsetRadios) = ["...."] };...` // hardcoded, but still changeable

Gives plenty of options with just 2 settings, if people really really want something custom they can change the gvar

Reply:
I like proposal from @PabstMirror. `GVAR` could also be made into a proper API function (maybe now, maybe later if there is a request).

Reply:
Fnc is there now, setter API in a future PR (maybeh)